pkg/cmd/server/crypto/crypto.go

```diff
@@ -206,14 +270,22 @@ func MakeCA(certFile, keyFile, serialFile, name string) (*CA, error) {
 		return nil, err
 	}
 
-	if err := ioutil.WriteFile(serialFile, []byte("0"), 0644); err != nil {
-		return nil, err
+	var serialGenerator SerialGenerator
+	if len(serialFile) > 0 {
+		if err := ioutil.WriteFile(serialFile, []byte("0"), 0644); err != nil {
```

Comment:
not new, but this should be `00` to match the "pad to 2 chars for openssl" bit below